trie/trieStorageManager.go

```diff
@@ -47,33 +44,25 @@ type snapshotsQueueEntry struct {
 
 // NewTrieStorageManagerArgs holds the arguments needed for creating a new trieStorageManager
 type NewTrieStorageManagerArgs struct {
-	MainStorer             common.BaseStorer
-	CheckpointsStorer      common.BaseStorer
-	Marshalizer            marshal.Marshalizer
-	Hasher                 hashing.Hasher
-	GeneralConfig          config.TrieStorageManagerConfig
-	CheckpointHashesHolder CheckpointHashesHolder
-	IdleProvider           IdleNodeProvider
-	Identifier             string
+	MainStorer    common.BaseStorer
+	Marshalizer   marshal.Marshalizer
+	Hasher        hashing.Hasher
+	GeneralConfig config.TrieStorageManagerConfig
+	IdleProvider  IdleNodeProvider
+	Identifier    string
 }
 
 // NewTrieStorageManager creates a new instance of trieStorageManager
 func NewTrieStorageManager(args NewTrieStorageManagerArgs) (*trieStorageManager, error) {
 	if check.IfNil(args.MainStorer) {
 		return nil, fmt.Errorf("%w for main storer", ErrNilStorer)
 	}
-	if check.IfNil(args.CheckpointsStorer) {
-		return nil, fmt.Errorf("%w for checkpoints storer", ErrNilStorer)
-	}
 	if check.IfNil(args.Marshalizer) {
 		return nil, ErrNilMarshalizer
 	}
 	if check.IfNil(args.Hasher) {
 		return nil, ErrNilHasher
 	}
-	if check.IfNil(args.CheckpointHashesHolder) {
-		return nil, ErrNilCheckpointHashesHolder
```

Comment:
i think `ErrNilCheckpointHashesHolder` has to be removed as well

Reply:
Done.